tests/src/providers/testqgsogrprovider.cpp

```diff
@@ -184,6 +184,12 @@ void failOnWarning( QtMsgType type, const QMessageLogContext &context, const QSt
 
 void TestQgsOgrProvider::testThread()
 {
+  // Disabled by @m-kuhn
+  // This test is flaky
+  // See https://travis-ci.org/qgis/QGIS/jobs/505008602#L6464-L7108
+  return;
```

Comment:
Why not just disable on Travis? There's a utility function in the test library which detects this

Reply:
Thanks